src/main/java/ladder/type/DirectionType.java

```diff
@@ -0,0 +1,20 @@
+package ladder.type;
+
+import java.util.function.Function;
+
+public enum DirectionType {
+
+	LEFT(i -> i - 1),
+	DOWN(i -> i),
+	RIGHT(i -> i + 1);
+
+	private final Function<Integer, Integer> move;
```

Comment:
```suggestion
	private final IntUnaryOperator move;
```

인수타입과 반환타입이 같다면 UnaryOperator를 활용하면 좋습니다 ㅎ
그리고 그게 primitive타입이 될 수있다면 위와 같이 제공되는 IntUnaryOperator등을 사용하면 내부적으로 primitive타입으로 해결하기에 성능도 더 좋습니다